plugins/content_distribution/providers/cortex/lib/api/KalturaCortexApiDistributionJobProviderData.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * @package plugins.cortexApiDistribution
+ * @subpackage api.objects
+ */
+class KalturaCortexApiDistributionJobProviderData extends KalturaConfigurableDistributionJobProviderData
+{
+	/**
+	 * @var string
+	 */
+	public $videoAssetFilePath;
+	
+	/**
+	 * @var string
+	 */
+	public $thumbAssetFilePath;
+
+	/**
+	 * @var KalturaCortexApiCaptionDistributionInfoArray
+	 */
+	public $captionsInfo;
+
+	public function __construct(KalturaDistributionJobData $distributionJobData = null)
+	{
+		parent::__construct($distributionJobData);
+	    
+		if(!$distributionJobData)
+			return;
+		
+		if(!($distributionJobData->distributionProfile instanceof KalturaCortexApiDistributionProfile))
+			return;
+			
+		$flavorAssets = assetPeer::retrieveByIds(explode(',', $distributionJobData->entryDistribution->flavorAssetIds));
+		if(count($flavorAssets)) // if we have specific flavor assets for this distribution, grab the first one
+			$flavorAsset = reset($flavorAssets);
+		else // take the source asset
+			$flavorAsset = assetPeer::retrieveOriginalReadyByEntryId($distributionJobData->entryDistribution->entryId);
+		
+		if($flavorAsset) 
+		{
+			$syncKey = $flavorAsset->getSyncKey(flavorAsset::FILE_SYNC_FLAVOR_ASSET_SUB_TYPE_ASSET);
+			if(kFileSyncUtils::fileSync_exists($syncKey))
+			{
+				$this->videoAssetFilePath = kFileSyncUtils::getLocalFilePathForKey($syncKey, false);
+			}
+		}
+		
+		$thumbAssets = assetPeer::retrieveByIds(explode(',', $distributionJobData->entryDistribution->thumbAssetIds));
+		if(count($thumbAssets))
+		{
+			$syncKey = reset($thumbAssets)->getSyncKey(thumbAsset::FILE_SYNC_FLAVOR_ASSET_SUB_TYPE_ASSET);
+			if(kFileSyncUtils::fileSync_exists($syncKey))
```

Comment:
you are not consistent with parentheses, the if/else should be with {}

Reply:
@inbal-ben-david Youre completely right:), here are the fixes - https://github.com/kaltura/server/pull/12050/commits/8f8542300a1883a8d625150a5fade75dd0408b44